actors/builtin/multisig/multisig_test.go

```diff
@@ -1443,6 +1357,65 @@ func TestRemoveSigner(t *testing.T) {
 			rt.Verify()
 		})
 	}
+
+	t.Run("remove signer removes approvals", func(t *testing.T) {
+		rt := builder.Build(t)
+		rt.AddIDAddress(anneNonID, anne)
+		signers := []addr.Address{anne, bob, chuck}
+
+		actor.constructAndVerify(rt, 3, noUnlockDuration, startEpoch, signers...)
+
+		// Anne proposes a tx
+		rt.SetCaller(anne, builtin.AccountActorCodeID)
+		actor.proposeOK(rt, chuck, big.Zero(), builtin.MethodSend, nil, nil)
+		// Bob approves
+		rt.SetCaller(bob, builtin.AccountActorCodeID)
+		actor.approveOK(rt, 0, nil, nil)
+
+		// Bob proposes a tx
+		actor.proposeOK(rt, chuck, big.Zero(), builtin.MethodSend, nil, nil)
+		// Anne approves
+		rt.SetCaller(anne, builtin.AccountActorCodeID)
+		actor.approveOK(rt, 1, nil, nil)
+
+		// Anne is removed, threshold dropped to 2 of 2
+		rt.SetCaller(receiver, builtin.MultisigActorCodeID)
+		actor.removeSigner(rt, anne, true)
+
+		// Anne's approval is removed from each transaction.
+		actor.assertTransactions(rt, multisig.Transaction{
+			To:       chuck,
+			Value:    big.Zero(),
+			Method:   builtin.MethodSend,
+			Params:   nil,
+			Approved: []addr.Address{bob},
+		}, multisig.Transaction{
+			To:       chuck,
+			Value:    big.Zero(),
+			Method:   builtin.MethodSend,
+			Params:   nil,
+			Approved: []addr.Address{bob},
+		})
+	})
+
+	t.Run("remove signer deletes solo proposals", func(t *testing.T) {
+		rt := builder.Build(t)
+		rt.AddIDAddress(anneNonID, anne)
```

Comment:
nit: this and the above appear unnecessary because you call `removeSigner` on `anne` not `anneNonID`

Reply:
Thanks, I changed this one to use `anneNonID`.